packages/block-library/src/columns/transforms.js

```diff
@@ -105,6 +105,16 @@ const transforms = {
 			},
 		},
 	],
+	to: [
+		{
+			type: 'block',
+			blocks: [ '*' ],
+			transform: ( attributes, innerBlocks ) =>
+				innerBlocks
+					.map( ( innerBlock ) => innerBlock.innerBlocks )
+					.flat(),
```

Comment:
Would `flatMap` help here?

Reply:
Totally forgot that was a thing, thank-you! I've [opened a follow-up](https://github.com/WordPress/gutenberg/pull/45684).

Reply:
Sorry I didn't give you a chance 😆